src/NServiceBus.RabbitMQ/Receiving/ModelExtensions.cs

```diff
@@ -22,13 +23,22 @@ class MessageState
                 },
                 scheduler);
 
-        public static Task BasicRejectAndRequeue(this IModel channel, ulong deliveryTag, TaskScheduler scheduler) =>
+        public static Task BasicRejectAndRequeueIfOpen(this IModel channel, ulong deliveryTag, TaskScheduler scheduler) =>
             TaskEx.StartNew(
                 new MessageState { Channel = channel, DeliveryTag = deliveryTag },
                 state =>
                 {
                     var messageState = (MessageState)state;
-                    messageState.Channel.BasicReject(messageState.DeliveryTag, true);
+                    if (channel.IsOpen)
```

Comment:
This needs to be `if (messageState.Channel.IsOpen)` to avoid capturing `channel`.

While you're there, could you add an empty line before the `if`?